jira_integration_example/utilities/alert.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+
+from google.cloud import monitoring_v3
+from google.protobuf.duration_pb2 import Duration
+from google.api_core.exceptions import NotFound
+
+
+PROJECT_ID = os.environ['GOOGLE_CLOUD_PROJECT']
```

Comment:
In my example, I suggested moving this into the test and CLI modules, respectively. I think it makes the required environments variables to use each script more obvious, since it's not buried in a dependency.